src/types/markers.js

```diff
@@ -697,7 +698,7 @@ export type MarkerPayload_Gecko =
   | GPUMarkerPayload
   | NetworkPayload
   | UserTimingMarkerPayload
-  | TextMarkerPayload
+  | $ReplaceCauseWithStack<TextMarkerPayload>
```

Comment:
My earlier comment also applies to this.
Also could you move this type to below? We have a comment there that explains the `$ReplaceCauseWithStack`, and all the payloads that have `cause` is grouped there.